docker/Dockerfile

```diff
@@ -113,6 +113,13 @@ RUN wget https://github.com/SimonKagstrom/kcov/archive/v36.tar.gz && \
 COPY --chown=root:root install-bolt.sh /tmp
 RUN /bin/bash /tmp/install-bolt.sh
 
+# Install grpcurl
+RUN mkdir -p /tmp/grpcurl
+WORKDIR /tmp/grpcurl
+RUN wget https://github.com/fullstorydev/grpcurl/releases/download/v1.7.0/grpcurl_1.7.0_linux_x86_64.tar.gz && \
+    tar xvf grpcurl_1.7.0_linux_x86_64.tar.gz && \
+    mv grpcurl /usr/bin/
```

Comment:
Do you still have to do things like set the GOPATH etc? Maybe a very short example usage, even just as a comment on this PR

Reply:
i don't think i need to do that -- I think it's a self-contained binary and doesn't depend on go being installed, and i didn't have to do that in my experiments yet, in my local env or in the mob prompt itself